assets/sass/components/_badges-tags.scss

```diff
@@ -3,11 +3,15 @@
     font-weight: normal;
   }
 
-  .govuk-tag.govuk-tag--red {
-    width: min-content;
+  .govuk-tag.govuk-tag--red, .govuk-tag--purple {
+    // width: min-content;
   }
 
   .govuk-tag--grey {
     white-space: nowrap;
   }
 }
+
```

Comment:
Can we remove these 4 empty lines? I'll see if there's a linting rule we can turn on that'll fix this automatically.